public/css/app.css

```diff
@@ -95,7 +95,7 @@ textarea::placeholder {
     pointer-events: none;
 }
 
-.sidebar-item-content > div > *:last-child {
+.sidebar-item-content>div>*:last-child {
```

Comment:
Is this part of a style convention or is it just your editor having its way again?